tox.ini

```diff
@@ -6,6 +6,10 @@ ignore = W503
 skipsdist = True
 envlist = lint,unit,integration
 
+[pytest]
+filterwarnings =
+    ignore::DeprecationWarning
+
```

Comment:
This should [no longer be necessary](https://github.com/charmed-kubernetes/pytest-operator/blob/main/pytest_operator/plugin.py#L52-L60).